e2e/specs/stateless/profileEditor.spec.ts

```diff
@@ -914,3 +914,84 @@ test.describe('subgraph errors', () => {
     await page.getByTestId('subgraph-network-error').uncheck()
   })
 })
+
+test('Should show edit profile button if user is manager', async ({
+  login,
+  makeName,
+  makePageObject,
+}) => {
+  const name = await makeName({
+    label: 'unwrapped',
+    type: 'legacy',
+    owner: 'user',
+    manager: 'user2',
+  })
+
+  const profilePage = makePageObject('ProfilePage')
+
+  await profilePage.goto(name)
+  await login.connect('user2')
+
```

Comment:
Hi @storywithoutend,

The test case is "Should show edit profile button if user is manager". However, I've noticed in the makeName setup that the owner is defined as "user" and the manager as "user2". If I use login.connect(), I assume this will automatically assign user as the owner. If this is correct, then this test case would essentially be identical to "Should show disabled edit profile button if user is owner but not manager". 

What are your thoughts?

Reply:
You can change the *makeName* setup to set **manager** to *user* and **owner** to *user2*.